### DISCUSSION TITLE
Demande d'aide au niveau de GitHub Pages

### DISCUSSION THREAD

**Discussion by John-Laguerre** [community/community#64879] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Bonjour à tous, 

Actuellement en études de développement Web, lors de la construction de mon quatrième projet et finalisation du projet, je souhaitais publier mon site finaliser sur GitHub Pages mais en le publiant, je n'ai pas le style qui s'affiche. J'ai essayé à plusieurs reprises de débug ce problème est trouvée par des moyens de recherche le pourquoi du comment mais je n'arrive toujours pas à trouver la solution au problème.  Pouvez-vous m'aider s'il vous plaît ?  merci d'avance.

**Comment by queenofcorgis** [comment]
Hi @John-Laguerre , thanks for participating!
We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
